benchmark/largerandom/ondemand.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#ifdef SIMDJSON_IMPLEMENTATION
+#if SIMDJSON_EXCEPTIONS
+
+#include "largerandom.h"
+
+namespace largerandom {
+namespace {
+
+using namespace simdjson;
+using namespace SIMDJSON_IMPLEMENTATION;
+using namespace SIMDJSON_IMPLEMENTATION::stage2;
+
+class OnDemand {
```

Comment:
I do not mind, but note that you are breaking your very own style conventions. 

OnDemand vs on_demand.

(This is not a request for changes.)

Reply:
Yeah. The reason I'm doing it is it seems to be Google Benchmark convention. I don't know what to choose here :)